.github/workflows/docs.yml

```
@@ -28,6 +28,7 @@ jobs:
         pip install --upgrade nox
         nox -s docs
     - name: Publish
+      if: github.repository == 'lundberg/respx'
```

Not a strong opinion, byt it might be cleaner to just condition on owner, what do you think?

```suggestion
      if: github.repository_owner == 'lundberg'
```

---

Yes, this looks cleaner.

If you have a similar thing in other repos then you can diff between your repos and this won't show as a change.

(I do have a similar thing in some of mine, so I'll change those like this too! Thanks!)